resources/assets/components/CampaignTable/Header.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+class Header extends React.Component {
+  render() {
+    return (
+      <thead>
+        <tr className="table__header">
```

Comment:
oh coool. I definitely wanted a more general Table component but wasn't sure how to approach it. I will try this!